python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -483,17 +476,27 @@ def _filter_by_project(self, actions, sw_entities, project):
         filtered = []
 
         for action in actions:
+            # The engine_name property of an engine command is defined by
+            # tk-multi-launchapp, and corresponds to the engine that provided
+            # the information necessary to register the launcher. If the action
```

Comment:
I completely misunderstood where `actions` was coming from. I thought you were iterating on the settings coming from the yml file. actions != env settings.
Sorry!!!